### ISSUE TITLE
Accept password in interactive prompt

### ISSUE THREAD

**Issue by rshigapov-bhft** [ClickHouse/ch2rs#9]
Cleartext password argument can be logged in shell history.

Interactive prompt with suppressed echo is a more secure way to enter password.

**Comment by loyd** [comment]
I agree (although you can prepend space to avoid adding to history, although it only works in some places).

How is it usually implemented in CLI? We cannot run an interactive prompt if `-p` is missing because it means "no password required" now. `-p -` usually means "see stdin".

Would you happen to have any suggestions here?

**Comment by rshigapov-bhft** [comment]
For example, PostgreSQL [provides](https://www.postgresql.org/docs/current/app-psql.html#APP-PSQL-OPTION-PASSWORD) `--password` flag to force password prompt even if authentication method doesn't require a password.

If stdin is exclusively reserved for passwords then it will be convenient to use the tool in automation scripts, Ansible playbooks or to retrieve password from a password manager.

Space prepending is not convenient because there are many arguments to configure and one might need to run the command more then once.

**Comment by stdfox** [comment]
This can be implemented via structopt and environment variables. It is common practice for CI/CD to use environment variables.

Something like this:
```
#[structopt(short = "p", env = "CLICKHOUSE_PASSWORD", hide_env_values = true)]
pub password: Option<String>,
```

I suggest taking the username and password from the URL parameter (ex. http://default:default@localhost:8123 or CLICKHOUSE_URL env if set) first, but giving priority to individual settings (CLICKHOUSE_USERNAME / CLICKHOUSE_PASSWORD).
